fluss-connectors/fluss-connector-flink/src/main/java/com/alibaba/fluss/connector/flink/sink/FlinkTableSink.java

```diff
@@ -67,17 +68,21 @@ public class FlinkTableSink
     private boolean appliedUpdates = false;
     @Nullable private GenericRow deleteRow;
 
+    private final MergeEngine mergeEngine;
+
     public FlinkTableSink(
```

Comment:
Add validation to throw exception in class `FlinkTableSink` that update/delete/partial update is not supported in the version merge engine and a test to verify the exception should be thrown...